libsolidity/ast/Types.cpp

```diff
@@ -110,6 +110,15 @@ util::Result<TypePointers> transformParametersToExternal(TypePointers const& _pa
 	return transformed;
 }
 
+string typesToString(TypePointers const& _types, bool _short)
+{
+	return '(' + util::joinHumanReadable(
+				_types | ranges::views::transform([&](auto const* _type) { return _type->toString(_short); }),
+				","
+			) +
+		')';
```

Comment:
```suggestion
			) + ')';
```
Either *always* break or *never* break is what the coding style says :-).
Also looking at this again, the indentation is off... it's multiple levels of indentation while it should only be one...

Reply:
Suggestion applied.